lib/mayaUsd/ufe/UsdTransform3d.cpp

```diff
@@ -60,17 +60,26 @@ namespace {
 	}
 }
 
-UsdTransform3d::UsdTransform3d()
-	: Transform3d()
+UsdTransform3d::UsdTransform3d() : Transform3d()
 {
 }
 
+UsdTransform3d::UsdTransform3d(const UsdSceneItem::Ptr& item)
```

Comment:
Had to go through setItem() because the default constructor did not take an argument, inconvenient, fixed.